import_ldraw.py

```diff
@@ -156,6 +162,7 @@ def __init__(self, context, filename, mat, colour=None):
         self.part_count = 0
 
         self.mat = mat
+        self.orientation = orientation
```

Comment:
Aha! Move this above `self.mat`, move the comment above this, then put a blank space between it and `self.mat`.